src/cockpit/transports.py

```diff
@@ -394,6 +404,7 @@ class StdioTransport(_Transport):
         - character devices (including terminals)
         - sockets
     """
+
```

Comment:
Given how the previous PR made it past CI, this can only mean that we don't cover src/python/ in our tests/static-code?

Reply:
I can't get any of the tools to complain about this.  I think it'll accept it one way or the other.

In any case, PEP 257 is fairly clear about what's correct:

> Insert a blank line after all docstrings (one-line or multi-line) that document a class – generally speaking, the class’s methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.

Reply:
Ah, I looked wrong. I thought this was a missing empty line before a function declaration. But of course it's just bewetween docstring and body.